internal/exec/terraform_outputs.go

```diff
@@ -321,21 +318,15 @@ func GetTerraformOutput(
 		// Execute `terraform output`
 		terraformOutputs, err := execTerraformOutput(atmosConfig, component, stack, sections)
 		if err != nil {
-			if atmosConfig.Logs.Level == u.LogLevelTrace || atmosConfig.Logs.Level == u.LogLevelDebug {
-				fmt.Printf("\r✗ %s\n", message)
-			}
-			l.Fatal("Failed to execute terraform output", "component", component, "stack", stack, "error", err)
+			u.PrintMessage(fmt.Sprintf("\r✗ %s\n", message))
```

Comment:
Does this go to stderr?

Reply:
Does this go to stderr?

https://github.com/cloudposse/atmos/blob/main/tests/test-cases/atmos-functions.yaml#L21-L26
Based on above test case we do not add the line to stderr.

Reply:
Done